src/controllers/midi/midicontroller.cpp

```diff
@@ -631,6 +631,12 @@ QJSValue MidiController::makeInputHandler(int status, int midino, const QJSValue
         return QJSValue();
     }
 
+    if (m_pMapping->getInputMappings().contains(status)) {
+        qDebug(m_logBase).nospace() << "Ignoring anonymous JS function for status "
+                                    << status << " because a previous binding exists";
```

Comment:
Here, I just log a message. I was thinking maybe also log the script line. `getScriptEngine()->throwJSError()` indicates a line in the script. Is there a similar method to just log a warning?

Reply:
My guess is: 
`getScriptEngine()->currentStackFrame->lineNumber()`

Reply:
that doesn't exists / is private, or at least it seems to be undocumented. where is this from?

Reply:
I have skimmed through the Qt source.

Reply:
I can't find `getScriptEngine()->currentStackFrame`.